src/domain/entities/alert/AlertData.ts

```diff
@@ -0,0 +1,23 @@
+import { Maybe } from "../../../utils/ts-utils";
+import { Id } from "../Ref";
+
+export type OutbreakType = "disease" | "hazard";
```

Comment:
as mentioned in earlier comment, instead of OutbreakType lets reuse Datasource which already exists.